src/app/alert-box/alert-box.component.less

```diff
@@ -1,82 +1,98 @@
+@headHeight: 40px;
+.mixin-headHeight {
+    font-size: 16px;
+    min-height: @headHeight;
+    max-height: @headHeight;
+    line-height: @headHeight;
+}
+.margin5LeftRight {
+    margin: 0 5px;
+}
 .alert-box {
-    // background: #fff;
-    text-align: left;
-    position: relative;
-    .alert-heading {
-        border-bottom: 1px solid #d1d1d1;
-        .alert-title {
-            // font-family: OpenSans-Semibold;
-            font-size: 16px;
-            color:#000;
-            #symbol {
-                padding-right: 4px;
-            }
-            #font-opensans {
-                font-weight: 600;;
+    .alert-head {
+        .mixin-headHeight;
+        &:after {
+            content: '';
+            display: block;
+            height: 1px;
+            background: #d1d1d1;
+            width: 100%;
+            top: 5px;
+            position: relative;
+        }
+        .icon-holder {
+            width: 20px;
+        }
+        & > span,
+        & > div {
+            float: left;
+        }
+        &:after {
```

Comment:
you can move these styles to the above `&:after` block

Reply:
Yes, that's a mistake. Thanks for pointing out!

Reply:
NP